src/manifest.json

```diff
@@ -93,6 +93,8 @@
     }
   ],
 
+  "content_security_policy": "object-src data: 'unsafe-eval'",
```

Comment:
@Vinnl @groovecoder I added this to fix [MPP-3607](https://mozilla-hub.atlassian.net/browse/MPP-3607) to allow this bit of the icon injection script: https://github.com/mozilla/fx-private-relay-add-on/blob/main/src/js/other-websites/icon.js#L69-L70 

Is this unsafe? What are our options?

Reply:
My two cents: If the SVG image source (iconUrl in your code) were to be constructed dynamically based on user input or untrusted data, it could be vulnerable to data injection attacks like Cross-Site Scripting (XSS). An attacker could potentially inject malicious code into the SVG data, leading to security vulnerabilities. That is not the case here. 

Maybe @groovecoder or @Vinnl can come up with another reason (or set of them).

Reply:
For the actual risks I'd defer to security experts (e.g. in `#security`), but as restrictive as possible should be safest, and as far as I can see, it can be quite a bit tighter than this? Specifically, I don't see why the referenced line would need `'unsafe-eval'`, and I'd expect [`img-src`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/img-src) rather than `object-src`?

And since we already know what image we'll be injecting, for bonus points I think you should even be able to [specify the image's hash](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy#other_values), rather than just allowing all `data:` URLs. (But when the CSP is restricted to just images, I think the main danger is that malicious scripts could inject misleading images if our extension is installed, which seems fairly manageable.)

Reply:
Welp, it wasn't loading on that page before the CSP policy, and it was loading after it. I'll play with some other policies (including the hash method too) to see how tightly I can get it to still work.

Reply:
Update: I removed it and it's now _working_. 🤔

Reply:
Ugh. Okay - So I refactored the add-on to use [`browser.runtime.getURL(path/to/icon.svg)`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/getURL) to get CSP-save/declared path and inject that image. 

However, I don't think this will work. See more in Jira: https://mozilla-hub.atlassian.net/browse/MPP-3607

Reply:
Agree with what @lloan and @Vinnl said that it's best to find the tightest possible CSP. I also couldn't reproduce the bug that says the icon is NOT showing up on Mozilla's Auth0 SSO page in Mac Firefox, so it looks like this is a Chrome-only issue. The `getURL` technique seems right, but good catch that it would expose the extension's ID to the page, which is a potential privacy leak.

The only fix I can think of would be to modify the `Content-Security-Policy` HTTP response header of the web page itself to allow inline `background-image`s - i.e., NOT this `content_security_policy` in the add-on's manifest. And we shouldn't modify the CSP of all web pages to allow unsafe inline images just to make our add-on icon show up!

So yeah, I think we can remove this line, and we have to just leave the image broken on some pages.

```suggestion
```